mod/thewire/views/default/resources/thewire/edit.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Edit page
+ */
+
+$enable_editing = (bool) elgg_get_plugin_setting('enable_editing', 'thewire');
+	
+if(!$enable_editing) {
+	throw new \Elgg\EntityPermissionsException();
+}
+
+$guid = elgg_extract('guid', $vars);
+elgg_entity_gatekeeper($guid, 'object', 'thewire');
+
+/* @var $post ElggWire */
+$post = get_entity($guid);
+
+elgg_push_entity_breadcrumbs($post, true);
+
+$content = elgg_view_form('thewire/add', [
+	'class' => 'thewire-form',
+	'prevent_double_submit' => true,
+], [
+	'guid' => $guid,
```

Comment:
I would suggest passing `'entity' => $entity` this will give more flexibility in the future